easybuild/easyconfigs/g/GCCcore/GCCcore-12.1.0.eb

```diff
@@ -28,7 +28,7 @@ sources = [
     'mpc-1.2.1.tar.gz',
     'isl-0.24.tar.bz2',
     'newlib-4.1.0.tar.gz',
-    {'download_filename': '7292758.tar.gz', 'filename': 'nvptx-tools-20220412.tar.gz'},
+    {'download_filename': '7292758.tar.gz', 'filename': 'nvptx-tools-20240419.tar.gz'},
```

Comment:
Unfortunately this doesn't actually fix anything. The `7292758` refers to a git hash; you need to change that too to get the version from 20240419. Same for the other easyconfigs here.